src/api-service/__app__/onefuzzlib/agent_authorization.py

```diff
@@ -54,6 +55,8 @@ def try_get_token_auth_header(request: func.HttpRequest) -> Union[Error, TokenDa
 
 @cached(ttl=60)
 def is_authorized(token_data: TokenData) -> bool:
```

Comment:
Not new to this PR, but let's add a comment emphasizing why we trust the `object_id` in the `TokenData`.

Reply:
Already have a comment on line 50 for that